GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/random.cpp

```diff
@@ -234,19 +234,19 @@ int Random2Class::operator() (int minval, int maxval)
 **	and thus prevents the algorithm from breaking down as a result.
 */
 int Random3Class::Mix1[20] = {
-	0x0baa96887, 0x01e17d32c, 0x003bcdc3c, 0x00f33d1b2,
-	0x076a6491d, 0x0c570d85d, 0x0e382b1e3, 0x078db4362,
-	0x07439a9d4, 0x09cea8ac5, 0x089537c5c, 0x02588f55d,
-	0x0415b5e1d, 0x0216e3d95, 0x085c662e7, 0x05e8ab368,
-	0x03ea5cc8c, 0x0d26a0f74, 0x0f3a9222b, 0x048aad7e4
+	static_cast<int>(0xbaa96887), static_cast<int>(0x1e17d32c), static_cast<int>(0x03bcdc3c), static_cast<int>(0x0f33d1b2),
```

Comment:
The first commit uses different wording style than the other commits.

It should say

"[ZH] Fix compiler warnings for [...]"